## Raise a GraphQL field error in case of a datasource invocation error
#if($ctx.error)
    $util.error($ctx.error.message, $ctx.error.type)
#end
## Pass back the result from DynamoDB. **
$util.qr($ctx.stash.put("DONOR QUERY RESULT", $util.toJson($ctx.result)))
#set($ctx.args.input.donorName = $ctx.result.donorName)
$util.toJson($ctx.result)